mappings/net/minecraft/screen/ScreenHandler.mapping

```diff
@@ -81,6 +81,10 @@ CLASS net/minecraft/class_1703 net/minecraft/screen/ScreenHandler
 	METHOD method_34715 checkPropertyUpdates (II)V
 		ARG 1 id
 		ARG 2 value
+	METHOD method_37419 notifyPropertyUpdate (II)V
+		ARG 1 index
+		ARG 2 value
+	METHOD method_37420 update ()V
```

Comment:
Just curious, how does this update work?

Reply:
It iterates over the slots, calls `updateTrackedSlot` for each one, then iterates over the properties, if it has changed calls `notifyPropertyUpdate`, which calls `onPropertyUpdate` for each listener

Reply:
is this like only called on client-side screen handlers on server-sent packets then? worth to note if that's the case

Reply:
Just checked, IDEA only found a usage of it in `ServerPlayNetworkHandler#onSlotClicked`, so it's only server side

Reply:
Should this be `updateToClient` or `sendToClient` to clarify? otherwise "update" sounds a bit too generic.